hw/xbox/xid-sdl.c

```diff
@@ -105,11 +109,12 @@ typedef struct USBXIDState {
     XIDGamepadOutputReport out_state_capabilities;
 
     uint8_t device_index;
+
+    HapticEmulationMode haptic_emulation_mode;
```

Comment:
nit: Let's drop the blank lines around this new field

Reply:
I felt like the other fields were more grouped by `sdl_` whereas this is a unique thing from that which is why I added the spacing. I'm not against moving it down though, I was on the fence about it anyway.